src/pages/transaction/Transaction.tsx

```diff
@@ -652,6 +591,86 @@ const Transaction: React.FC<Props> = (props: Props) => {
+                        value={
+                          !isLoading && transaction
+                            ? transaction.witnesses[0].invocation
+                            : ''
+                        }
+                        withCopyButton
+                      />
+                      <TransactionBlock
+                        label="VERIFICATION SCRIPT"
+                        value={
+                          !isLoading && transaction
+                            ? transaction.witnesses[0].verification
+                            : ''
+                        }
+                        withCopyButton
+                      />
+                      <TransactionBlock
+                        label="SCRIPT"
+                        value={
+                          !isLoading && transaction ? transaction.script : ''
+                        }
+                        withCopyButton
+                      />
+                    </div>
+                  </ExpandingPanel>
+                )}
+
+              {transaction &&
+                transaction.witnesses &&
+                transaction.witnesses[0] && (
+                  <div style={{ margin: '24px 0' }}>
+                    <ExpandingPanel title="DISASSEMBLED SCRIPT" open={false}>
+                      <div className="script-tile-row">
+                        <TransactionBlock
+                          label="INVOCATION SCRIPT"
+                          value={
+                            !isLoading && transaction
+                              ? neo3Disassemble(
+                                  transaction.witnesses[0].invocation,
+                                )
+                              : ''
+                          }
+                          withCopyButton
+                        />
+
+                        <TransactionBlock
+                          label="VERIFICATION SCRIPT"
+                          value={
+                            !isLoading &&
+                            transaction &&
+                            transaction.witnesses &&
+                            transaction.witnesses[0]
+                              ? neo3Disassemble(
+                                  transaction.witnesses[0].verification,
+                                )
+                              : ''
+                          }
+                          withCopyButton
+                        />
+                        <TransactionBlock
+                          label="SCRIPT"
+                          value={
+                            !isLoading && transaction && transaction.script
+                              ? neo3Disassemble(transaction.script)
+                              : ''
+                          }
+                          withCopyButton
+                        />
+                      </div>
+                    </ExpandingPanel>
+                  </div>
+                )}
```

Comment:
Can you fragment this component into small others? This component has many lines.